### DISCUSSION TITLE
wie scannt die 7490 die andere WLAN-Funknetze in der Umgebung

### DISCUSSION THREAD

**Discussion by muckel-sassi** [Freetz-NG/freetz-ng#175] [Allgemein]
Hallo,
wie scannt eigentlich die 7490 die andere WLAN-Funknetze in der Umgebung?

iwlist ist weder im original build noch im modified enthalten?

Im AVM-WI werden aberr welche angezeigt....

**Comment by fda77** [comment]
Das kann dir hippie2000 bestimmt erklären. Er mag github aber nicht und ist im irc ("webchat") erreichbar: https://boxmatrix.info/wiki/IRC-Channels

**Comment by muckel-sassi** [comment]
Danke für die Antwort, müsste aber nicht iwlist im original build vorhanden sein?

**Reply by fda77** [reply]
Wenn das Wlan ohne die Datei funktioniert: scheinbar nicht

**Comment by muckel-sassi** [comment]
bei der 7590 ist iwlist mit drin.....

**Reply by fda77** [reply]
Nicht nur das, die hat sogar doppelt so viel RAM drin.
